tei/el-hombre-pobre-todo-es-trazas.xml

```diff
@@ -713,6 +713,9 @@
 
             <l>Adiós.</l>
           </sp>
+          <stage>Vase Don Juan.</stage>
+        </div>
+        <div type="scene" resp="#rojas" n="2">
```

Comment:
Since this is a scene _not_ added by Antonio, you may want to either remove the `resp` attribute or reference yourself.

Reply:
I'd suggest that @arojascastro just reviews my change. If he agrees with it, he can take resposibility for it and we can keep the attribute value as is.

Reply:
I tried to add @arojascastro as a reviewer but couldn't, probably because he hadn't been added as a collaborator to the repo yet. Maybe you can ask him explicitly in the other PRs to have a look.